### ISSUE TITLE
Version 1.1.4 not found in JCenter nor in MavenCentral

### ISSUE THREAD

**Issue by orgmir** [zserge/log#5]
I want to use the latest version in my project (that has the Printer interface) and when syncing Android Studio, gradle project refresh fails with:

`Error:Cause: inconsistent module metadata found. Descriptor: co.trikita:log:1.1.1 Errors: bad version: expected='1.1.4' found='1.1.1'`

My build.gradle:

```
dependencies {
  (...)
  compile 'co.trikita:log:1.1.4'
}
repositories {
 (...)
  mavenCentral()
  jcenter()
}
```

I have copy pasted the file to my project, but I would prefer using gradle to keep it up to date :)

**Comment by zserge** [comment]
Interesting. I can clearly see the 1.1.4 version on maven: https://repo1.maven.org/maven2/co/trikita/log/1.1.4/

I'm not sure how to check it manually on jcenter(), but I see version 1.1.4 in my list.

**Comment by zserge** [comment]
Oh, I see. In the POM XML version is still 1.1.1. Seems like 1.1.5 is on the way :)

**Comment by zserge** [comment]
@Orgmir Ok, I tried to re-publish 1.1.4 and it seems to work this time. Can you please try again?
If not hard - can you please try both, jcenter and mavenCentral?

**Comment by zserge** [comment]
Released version 1.1.5. It should be available on both Maven and Bintray now. If you still experience problems with downloading it - please reopen this issue.

**Comment by orgmir** [comment]
Sorry for the delay, I messed the github notifications for mentions. I'm now using 1.1.5 and everything is working :+1:
